File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        stack = [] 
        output = [0] * len(temperatures)
        
        for i, temp in enumerate(temperatures):
            while stack and temp > temperatures[stack[-1]]:
                idx = stack.pop()
                output[idx] = i - idx
            stack.append(i)
        return output   

File: Data Structures & Algorithms/is-subsequence/submission-0.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int i = 0;
        int j = 0;
        String temp = "";

        while (i < s.length() && j < t.length()){
            char s1 = s.charAt(i);
            char t1 = t.charAt(j);
            if (s1 == t1){
                temp += s1;
                i++;
            }
            j++;
        }
        return s.equals(temp);
    }
}